portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,24 +35,37 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Youtube Idea").addSort("timestamp", SortDirection.ASCENDING);
 
-    // TODO:  Make ArrayList more abstract aka make it a List.
-    ArrayList<String> sampleContents = new ArrayList<String>();
-    sampleContents.add("My name is Miah");
-    sampleContents.add("Im from NYC");
-    sampleContents.add("I love coding!");
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    List<String> ideas = new ArrayList<>();
+    for (Entity entity: results.asIterable()) {
+        String idea = (String) entity.getProperty("idea");
+
+        ideas.add(idea);
+    }
+
+    Gson gson = new Gson();
```

Comment:
Lets actually also make this a private final instance variable, we should need only one Gson instance 
Check out the javadoc for the Gson class for why that is: https://www.javadoc.io/doc/com.google.code.gson/gson/latest/com.google.gson/com/google/gson/Gson.html 
(and also we can talk about reading javadocs at tomorrow's standup which is a good topic of discussion!) :)